config/jobs/kubernetes-sigs/kube-storage-version-migrator/kube-storage-version-migrator-ci.yaml

```diff
@@ -0,0 +1,31 @@
+presubmits:
+  kubernetes-sigs/kube-storage-version-migrator:
```

Comment:
Do you want this to appear in testgrid?

Reply:
Thanks for the reminder. Added this one and a couple of existing tests to testgrid/config.yaml. PTAL. Thank you @fejta !